homeassistant/helpers/schema_config_entry_flow.py

```diff
@@ -25,8 +25,13 @@ class SchemaFlowError(Exception):
 class SchemaFlowFormStep:
     """Define a config or options flow step."""
 
-    # Optional schema for requesting and validating user input. If schema validation
-    # fails, the step will be retried. If the schema is None, no user input is requested.
+    # Optional voluptuous schema, or function which returns a schema or None, for
+    # requesting and validating user input.
+    # If a function is specified, the function will be passed the handler, which is
+    # either an instance of SchemaConfigFlowHandler or SchemaOptionsFlowHandler, and the
+    # union of config entry options and user input from previous steps.
+    # If schema validation fails, the step will be retried. If the schema is None, no
+    # user input is requested.
```

Comment:
After-thought: could/should these be docstring instead of comments?

Reply:
yeah, they should indeed be docstrings.